.github/workflows/deploy-test.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   deploy-test:
-    if: ${{ github.event.label.name == 'test-deploy' }}`
+    if: ${{ github.event.label.name == 'test-deploy' }} || github.event.label.name !='integration'`
```

Comment:
why do we need the `||` condition?

Reply:
we don't want to build the test pypi if we added the integration label